content/en/docs/started/installing-kubeflow.md

```diff
@@ -5,30 +5,155 @@ weight = 20
+            Kubeflow Spark Operator
+          </a>
+        </td>
+        <td>
+          <a href="https://github.com/kubeflow/spark-operator">
+            <code>kubeflow/spark-operator</code>
+          </a>
+        </td>
+        <td>
+          Data Preparation
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <a href="/docs/components/katib/installation/#installing-katib">
+            Kubeflow Katib
+          </a>
+        </td>
+        <td>
+          <a href="https://github.com/kubeflow/katib">
+            <code>kubeflow/katib</code>
+          </a>
+        </td>
+        <td>
+          Model Optimization and AutoML
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <a href="/docs/components/training/installation/#installing-training-operator">
+            Kubeflow Training Operator
+          </a>
+        </td>
+        <td>
+          <a href="https://github.com/kubeflow/training-operator">
+            <code>kubeflow/training-operator</code>
+          </a>
+        </td>
+        <td>
+          Model Training and Fine-Tuning
+        </td>
+      </tr>
+      <tr>
+        <td>
+         <a href="/docs/components/training/user-guides/mpi/#installation">
+            Kubeflow MPI Operator
+          </a>
+        </td>
+        <td>
+          <a href="https://github.com/kubeflow/mpi-operator">
+            <code>kubeflow/mpi-operator</code>
+          </a>
+        </td>
+        <td>
+          All-Reduce Model Training
+        </td>
+      </tr>
+      <tr>
+        <td>
+         <a href="/docs/components/model-registry/installation/#installing-model-registry">
+            Kubeflow Model Registry
+          </a>
+        </td>
+        <td>
+          <a href="https://github.com/kubeflow/model-registry">
+            <code>kubeflow/model-registry</code>
+          </a>
+        </td>
+        <td>
+          Model Registry
+        </td>
+      </tr>
+      <tr>
+        <td>
+         <a href="https://kserve.github.io/website/master/admin/serverless/serverless">
+            KServe
+          </a>
+        </td>
+        <td>
+          <a href="https://github.com/kserve/kserve">
+            <code>kserve/kserve</code>
+          </a>
+        </td>
+        <td>
+          Model Serving
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <a href="docs/components/pipelines/v2/installation/quickstart/">
+            Kubeflow Pipelines
+          </a>
+        </td>
+        <td>
+          <a href="https://github.com/kubeflow/pipelines">
+            <code>kubeflow/pipelines</code>
+          </a>
+        </td>
+        <td>
+          ML Workflows and Schedulers
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+**Note**. Currently, Kubeflow Notebooks can't be deployed as a standalone application, but Notebooks
+WG is working on that as part of [this issue](https://github.com/kubeflow/kubeflow/issues/7549).
```

Comment:
As we discussed before with @kubeflow/kubeflow-steering-committee, this should be explained in the installation guide of individual Kubeflow components. Since we can't list all unavailable features in the installation section.

Pipelines, Notebooks can add those limitations in [the installation guides](https://www.kubeflow.org/docs/components/katib/installation/#installing-katib). It can introduce sub-section in the component installation guide: `Standalone Pipelines Limitations`. @StefanoFioravanzo @hbelmiro Any feedback ?

Reply:
Yes, we need to push this info down to the component's doc. @thesuperzapper  we can work together on enhancing the Notebooks docs, following recent approaches in Training Operator, Katib, Pipelines, Model Registry. You can see some examples here:

- https://github.com/kubeflow/training-operator/issues/1998
- https://github.com/kubeflow/katib/issues/2314
- https://github.com/kubeflow/website/issues/3716

Reply:
In the table, under the component links, I was only able to get the KServe and Spark Operator links to work.   The rest are broken.

Reply:
@andreyvelich we can mark this one as "resolved".

We can explain that Notebooks is only available in "Kubeflow Platform" under the `## Kubeflow Platform` heading as proposed in https://github.com/kubeflow/website/pull/3724/files#r1608938065